samase.py

```diff
@@ -1,29 +1,30 @@
 import os
 import subprocess
+import enum
 
 SAMASE = "bin\\samase.exe"
 SAMASE64 = "bin\\samase64.exe"
 
-def samase_line_constructor(operation_type:int, path:str, is64: bool = False):
+class OpType(enum.Enum):
```

Comment:
If we for some reason needed to stick with runtime dispatch over `OpType` (in a way more complex system, not really ever a case in a program like this) it's better to have it named instead of hardcoding magic values.